### PR TITLE
add average meter

### PR DESCRIPTION
**Pull request by ramasesh**
In `uv/meters`, added a meter which aggregates the average of several repeated measurements.

### CODE REVIEW

File: uv/meters/avg_meter.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class AverageMeter:
+
+  def __init__(self):
+    self.reset()
+
+  def reset(self):
+    self.val = 0
```
Comment (sritchie) [inline uv/meters/avg_meter.py:7]: you've stumbled on a powerful idea... aggregators! We built a whole zoo of these at https://github.com/twitter/algebird.

To do this right, you really want to take this note into account from `AveragedValue`:

https://github.com/twitter/algebird/blob/develop/algebird-core/src/main/scala/com/twitter/algebird/AveragedValue.scala#L29

```
 * The mean calculation uses a numerically stable online algorithm
 * suitable for large numbers of records, similar to Chan et. al.'s
 * [[http://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Parallel_algorithm
 * parallel variance algorithm on Wikipedia]]. As long as your count
 * doesn't overflow a Long, the mean calculation won't overflow.
```

This is the more numerically stable version: https://github.com/twitter/algebird/blob/develop/algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala#L98

Comment (sritchie) [reply uv/meters/avg_meter.py:7]: Keras has a bunch of these too: https://www.tensorflow.org/api_docs/python/tf/keras/metrics/Metric

`Sum`, `Mean` etc if you want some inspiration

**Comment by ramasesh** [comment]
Seems like this doesn't really fit with the rest of the functionality in `UV`, so I will close this now.
